src/dev/hmrServer.ts

```
@@ -0,0 +1,317 @@
+    return response;
+  };
+  
+  const handleRequest = async (req: Request): Promise<Response> => {
+    const url = new URL(req.url);
+    const pathname = url.pathname;
+    
+    try {
+      switch (pathname) {
+        case '/':
+        case '/html':
+          const htmlFile = await handleHTMLPageRequest('./example/build/html/pages/HtmlExample.html');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)